docs/jobs/openings/technical-writer.md

```diff
@@ -0,0 +1,31 @@
+# Documentation Technical Writer 
+
+Aragon is an open source, decentralized project [fighting for the freedom](https://www.youtube.com/watch?v=AqjIWmiAidw) to organize. We believe that decentralized organizations can solve the world's worst problems.
+
+We’re looking for a Technical Writer to help us communicate the different moving parts of [Aragon Core](https://app.aragon.org) and [Dev Tools](https://hack.aragon.org) to facilitate the development of the Ethereum community. Our Technical Writer will be the bridge between Aragon engineers and external developers, as well as the reference to support the growth of the Ethereum community through better documentation.
+
+### Responsibilities
+
+- Write, publish and maintain clear, concise, accurate and easy to understand developer documentation in the [Aragon Developer Portal](https://hack.aragon.org/).
+- Write, edit and publish technical articles and announcements.
```

Comment:
Would add links here for [Aragon Project blog](https://blog.aragon.org) and the [Aragon One blog](https://blog.aragon.one)